api/src/main/java/org/eclipse/microprofile/metrics/MetadataBuilder.java

```diff
@@ -54,7 +56,7 @@
     private Map<String, String> tags = new HashMap<>();
 
     MetadataBuilder() {
-        String globalTagsFromEnv = System.getenv(GLOBAL_TAGS_VARIABLE);
+        String globalTagsFromEnv = AccessController.doPrivileged((PrivilegedAction<String>) () -> System.getenv(GLOBAL_TAGS_VARIABLE));
```

Comment:
Just FYI that in master, this code will probably move to the new `MetricID` class through https://github.com/eclipse/microprofile-metrics/pull/312 so you might see some conflict soon